docs/README.md

```diff
@@ -0,0 +1,64 @@
+# HyperFormula documentation
+
+HyperFormula comes with a dedicated, regularly-updated documentation portal.
+
+View the documentation's latest production version at https://handsontable.github.io/hyperformula/.
+
+## About HyperFormula documentation
+
+The HyperFormula documentation is built with [VuePress](https://vuepress.vuejs.org/), a Vue-powered Static Site Generator.
+
+Our VuePress setup:
+
+* Publishes the docs as a sleek, responsive single-page application (SPA).
+* Boasts a great search engine.
+* Lets us write the docs in Markdown (and use Vue inside Markdown!).
+* Lets us quickly build the docs output onto a local server.
+* [COMING SOON] Versions and deploys the docs together with the software.
+
+## Getting started with HyperFormula documentation
+
+To start a local HyperFormula docs server:
+
+1. Get the right npm version:
+   * Later than 7.5
+   * Earlier than 16.0
+2. From the `hyperformula` directory, install the docs dependencies:
+    ```bash
+    npm install
+    ```
+3. From the `hyperformula` directory, start your local docs server:
+   ```bash
+   npm run docs:dev
+   ```
+4. In your browser, go to: http://localhost:8080/hyperformula/.
```

Comment:
Step missing: `npm run docs:api` before starting the server. Generate API reference from the HF source code.

And HF should be already build in the `/dist` directory: `npm bundle-all` I think. Otherwise you will get an error that `hyperformula.min.js` file was not found.